components/AirdropVerifier.vue

```diff
@@ -4,48 +4,77 @@
       'flex',
       'flex-col',
       'items-center',
+      'justify-center',
+      'px-[24px]',
     ]"
   >
     <img
-      :class="['my-[32px]', 'w-[300px]']"
+      :class="[
+        'hidden',
+        'my-[32px]',
+        'w-[300px]',
+        'lg:block',
+      ]"
       src="/images/airdrop/title_Checker.png"
     />
```

Comment:
Can be refactored along with the following?
https://github.com/likecoin/app-like-co/blob/a1d67dcf72e4f7f4438c096de72dc23f03368a59/components/AirdropLogin.vue#L12-L20